gen/definitions/profile_parcels/transport_cellular_profile.yaml

```diff
@@ -20,13 +20,13 @@ attributes:
   - model_name: id
     tf_name: profile_id
     data_path: [profileConfig]
-    mandatory: true
     example: 1
+    test_value: '1'
```

Comment:
`test_value` is only required if it is different than `example` value, otherwise it can be omitted.